easybuild/easyconfigs/c/CMake/CMake-3.9.4-foss-2017b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.9.4'
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
```

Comment:
Also here, no need for this easyconfig, since there already is an easyconfig using `GCCcore/6.4.0` toolchain (also for CMake 3.9.5 btw, in `develop` branch)